### ISSUE TITLE
select column meta for table map event

### ISSUE THREAD

**Issue by siddontang** [siddontang/go-mysql#32]
I think this can improve #28.

@karmakaze

**Comment by karmakaze** [comment]
Yes it does. However have to be mindful that a schema change may mean that the column meta is newer than the point in time of the stream that is being processed.

**Comment by siddontang** [comment]
We can check table id too, if table id exists, we can guarantee that the schema is not changed. if not, we may degrade to old use.

**Comment by gdey** [comment]
Is there a way to get the table id from the information_schema? I'm running into this issue as well, and want to know if the map_table_event corresponds to current table schema.

**Comment by siddontang** [comment]
Hi @gdey 
You can find it in `INFORMATION_SCHEMA.INNODB_SYS_TABLES`, but as you see, this may be only for innodb. :sob:
